src/platform/webos/Logging.cpp

```diff
@@ -0,0 +1,52 @@
+/* See Project CHIP LICENSE file for licensing information. */
```

Comment:
This license blurb is different fro the rest

Reply:
@andy31415 
I modified the license blurb of src/platform/webos/Logging.cpp just like other files.
Thank you for your review.